MANIFEST.in

```diff
@@ -58,6 +58,7 @@ exclude .github/ISSUE_TEMPLATE/feature_request.md
 exclude .github/PULL_REQUEST_TEMPLATE.md
 
 exclude mne.qrc
+recursive-exclude mne/images
 recursive-exclude mne/images *.svg
```

Comment:
There seems to be some duplication here, right? I would suggest to use `prune mne/images`

Reply:
Exactly what I wanted